CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 #### Minimum Required Rust
 
-* As of this release, `clap` requires `rustc 1.33.0` or greater.
+* As of this release, `clap` requires `rustc 1.36.0` or greater.
```

Comment:
It's theoretically a breaking change, but 1.33 if pretty old and it's unlikely that many people still use it

Reply:
It's not actually a breaking change since we hadn't released the 1.33 version bump. Also, in README, we say that Clap supports stable minus 2 versions.

Reply:
It is a breaking change. Though note we won't be releasing a new 2.0 version unless it's an emergency